package.json

```diff
@@ -69,6 +69,7 @@
     "remark-html": "^5.0.1",
     "request": "^2.79.0",
     "sinon": "^1.15.4",
+    "st": "^1.2.0",
```

Comment:
Ah.  Yeah, it wasn't working for me -- e.g. the `start-debug` script in the root package.json failed to find `st`.